### PR TITLE
completed basic functionality of square router

### LINKED ISSUE #18: Add API Routes for Squares

### PR DESCRIPTION
**Pull request by wjrhee**
closes #18

### CODE REVIEW

File: server/app/routes/squares/square.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          })
+      }
+    })
+})
+
+router.post('/', function(req, res, next){
+  return Square.create(req.body)
+    .then(function(square){
+      res.status(201).send(square);
+    })
```
Comment (Abazhenov) [inline server/app/routes/squares/square.js]: add a catch here

File: server/app/routes/squares/square.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  return Square.findById(req.params.id)
+    .then(function(square){
+      if(!square) res.sendStatus(404);
+      else{
+        square.update(req.body)
+          .then(function(updatedSquare){
+            res.json(updatedSquare);
+          })
+      }
+    })
```
Comment (Abazhenov) [inline server/app/routes/squares/square.js]: add a catch here

**Comment by Abazhenov** [comment]
Looks great just add the catches mentioned in line notes

**Comment by giocodes** [comment]
I'll work on the conflict

**Pull request merged**